code/game/mecha/mecha_parts.dm

```diff
@@ -10,6 +10,12 @@
 	flags = CONDUCT
 	origin_tech = "programming=2;materials=2;engineering=2"
 
+/obj/item/mecha_parts/core
+	name = "mech power core"
+	icon = 'icons/obj/assemblies/new_assemblies.dmi'
+	icon_state = "anomaly_core"
+	desc = "A complex piece of electronics used to regulate the large amounts of power used by combat mechs' delicate components."
```

Comment:
```suggestion
	desc = "A complex piece of electronics used to regulate the large amounts of power used by a combat mechs' delicate components."
```

Reply:
mech's, if you're going to slap an A in there.